server/runtime/src/main/java/org/infinispan/server/ExitStatus.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.server;
+
+import java.io.Serializable;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.0
+ **/
+public class ExitStatus implements Serializable {
```

Comment:
what are the plans for this class? this could be an `enum` since the internal state is never queried anywhere.

Reply:
My idea was to use the status as the value passed to a System.exit(...), but I need to refactor Bootstrap.java. Let's leave it like this for now.

Reply:
ok